webapp/src/components/App/Singlestudy/explore/Results/ResultDetails/ResultFilters.tsx

```diff
@@ -187,8 +187,12 @@ function ResultFilters({
       id: "search",
       label: "",
       field: (
-        <Box sx={{ width: 200 }}>
-          <SearchFE value={filters.search} onChange={(e) => handleSearchChange(e.target.value)} />
+        <Box sx={{ width: 150, mr: 1 }}>
```

Comment:
You can remove the width, the minWidth by default is 150 on focus.
And Box is not necessary.